core/iwasm/compilation/aot_compiler.c

```diff
@@ -361,13 +361,17 @@ aot_compile_func(AOTCompContext *comp_ctx, uint32 func_index)
 
                 read_leb_uint32(frame_ip, frame_ip_end, type_idx);
 
+                if (false
+#if WASM_ENABLE_GC != 0
+                    || comp_ctx->enable_gc
```

Comment:
enable_gc is always available in AOT compiler, no need to guard it with WASM_ENABLE_GC